packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -37,6 +37,10 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
   bytes32[] public assetAllocationSymbols;
   uint256[] public assetAllocationWeights;
 
+  uint256 private lastSpendingDay;
```

Comment:
`lastSpendingDay` & `spendingLimit` should be public, since a caller might want to check if the transfer will fail beforehand.

`spendingRatio` can be private, since we have a getter, which unwraps it.